codeguru_profiler_agent/utils/time.py

```diff
@@ -5,7 +5,7 @@
 
 def to_iso(epoch_milli):
     try:
-        return datetime.fromtimestamp(epoch_milli / 1000).isoformat()
+        return datetime.fromtimestamp(epoch_milli / 1000).isoformat() + "Z"  # ISO 8601 date-time format
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)

Reply:
Not a bad advice, it looks like the current failure on automated test is because a unit tests expects a timestamp of 1am GMT and got 1am PST. Please fix the epoch time used in the unit test.

Reply:
The allowed options for timestamp are:

1. "2020-12-23T19:33:18.878Z"
2. "2020-12-23T19:33:18Z"

Using `utcfromtimestamp` to make sure that the epoch is converted to utc time and that's what will be used to set as `agentStartTime` in the debug info of profile.